# Copyright (C) 2023 Light, Inc.
#
# This program is free software: you can redistribute it and/or modify
# it under the terms of the GNU Affero General Public License as published by
# the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.
#
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU Affero General Public License for more details.
#
# You should have received a copy of the GNU Affero General Public License
# along with this program.  If not, see <http://www.gnu.org/licenses/>.

diff --git a/dist/build/webpack/loaders/next-flight-loader/action-proxy.js b/dist/build/webpack/loaders/next-flight-loader/action-proxy.js
index 308995a3d78958f2ba58ff4166a3e03790125e6a..9bb0d609307c5c86a8d306c13f778c80af40eb3d 100644
--- a/dist/build/webpack/loaders/next-flight-loader/action-proxy.js
+++ b/dist/build/webpack/loaders/next-flight-loader/action-proxy.js
@@ -47,7 +47,7 @@ function createActionProxy(id, bound, action, originalAction) {
         bind: {
             value: bindImpl
         }
-    });
+    }, {configurable: true});
 }
 
 //# sourceMappingURL=action-proxy.js.map
\ No newline at end of file
diff --git a/dist/esm/build/webpack/loaders/next-flight-loader/action-proxy.js b/dist/esm/build/webpack/loaders/next-flight-loader/action-proxy.js
index 3b1ef848836bd2863e284d8eea581624056894da..899f266e527668b22c4e0f120c64863588c9375a 100644
--- a/dist/esm/build/webpack/loaders/next-flight-loader/action-proxy.js
+++ b/dist/esm/build/webpack/loaders/next-flight-loader/action-proxy.js
@@ -37,7 +37,7 @@ export function createActionProxy(id, bound, action, originalAction) {
         bind: {
             value: bindImpl
         }
-    });
+    }, {configurable: true});
 }
 
 //# sourceMappingURL=action-proxy.js.map